documentation/src/main.ts

```diff
@@ -13,7 +13,4 @@ governing permissions and limitations under the License.
 import polyfill from 'element-closest';
 
 polyfill(window);
-import('./components');
-import('./main.css');
-import('./router');
-import('./utils/spectrum');
+import('./app');
```

Comment:
Ensure these files can be _bundled_ as code that is loaded and run after the polyfill rather than three separate bundles, they are all needed for the first page load.